azerothcore/azerothcore-wotlk
# fix(Core/ICC): Intro Event Timers and Text

## CHANGES PROPOSED:
-  Change the first line of Tirion Fordring's text in ICC to correlate to the sound being heard (it had the first two lines of dialogue in one line)
-  Delay EVENT_BOLVAR_INTRO_1 so it doesn't overlap with EVENT_LK_INTRO_4


## ISSUES ADDRESSED:
- Tirion Fordring had two lines of text into one.
- Bolvar Fordragon's sound overlapped with The Lich King's sound

Current AC, Tirion says this in one big line of text:
Highlord Tirion Fordring yells: This is our final stand. What happens here will echo through the ages. Regardless of outcome, they will know that we fought with honor. That we fought for the freedom and safety of our people!  Remember, hero, fear is your greatest enemy in these befouled halls. Steel your heart and your soul will shine brighter than a thousand suns. The enemy will falter at the sight of you. They will fall as the light of righteousness envelops them!


Current AC, sound for the bolded text overlapped.:
The Lich King yells: The breaking of this one has been taxing. The atrocities I have committed upon his soul. He has resisted for so long, **but he will bow down before his king soon**.
Highlord Bolvar Fordragon yells: **NEVER! I... I will never... serve... you...**

## TESTS PERFORMED:
1. Applied SQL
2. Changed timer (icecrown_citadel.cpp)
3. Build successful
4. Startup, zone into ICC and initiated intro roleplay event

## HOW TO TEST THE CHANGES:
1. Apply SQL, build, launch server
2. Create character
3. .tele icecrowncitadel, .gm on and zone into ICC on any difficulty, either faction
4. Kill the first two "The Damned" to initiate roleplay
5. Pay close attention to Tirion's text in the beginning, it is now split properly between sound files
6. Wait towards the end, and listen for Lich King to finish saying "bow down before his king soon" - Bolvar no longer starts speaking before the Lich King finishes.


## Target branch(es):

Huge thanks to Barbz for walking me through my first PR :)
